lib/rspec/expectations/configuration.rb

```diff
@@ -44,6 +44,11 @@ def initialize
       #     end
       #   end
       def syntax=(values)
+        RSpec.deprecate(
```

Comment:
The issue with this, it prints a warning on loading a make test fail here https://github.com/rspec/rspec-support/blob/d04c8aef72d030fc354902f1e1b486b5c21e0732/lib/rspec/support/spec/library_wide_checks.rb#L108

Reply:
I went like [this](https://github.com/rspec/rspec-mocks/pull/1418/files#diff-dbe6c4ceed9ffa55bc69c698f87f0f0590bc8a20c46b3d8a10f6c42bb21baea9R214) in Core. We can remove this later on.

Reply:
Oh yes. I read it but forget about it. Sorry.